tools/nightly-e2e-tests/monitors-gen/.terraform.lock.hcl

```diff
@@ -0,0 +1,22 @@
+# This file is maintained automatically by "terraform init".
+# Manual edits may be lost in future updates.
+
+provider "registry.terraform.io/datadog/datadog" {
```

Comment:
just to confirm: this file should be commited?

Reply:
Like noticed in #521 this file will be removed. I will first merge #521 to this PR, so this file won't be committed to the main branch. I also made sure that there are no secrets inside before pushing it to remote 👍.